### PR TITLE
Implement Put, Remove and Scan of lifecycle.

### PR DESCRIPTION
**Pull request by sunfengyun**
Implement Put, Remove and Scan of lifecycle. 
Start lc with cache config in “yig.toml”.

### CODE REVIEW

**Comment by sunfengyun** [comment]
@dahefanteng  fmt deleted.

**Comment by thesues** [comment]
--- FAIL: Test_LifeCycle (3.62s)
	lifecycle_test.go:22: CreateBucket Success!
	lifecycle_test.go:29: PutObject Success!
	lifecycle_test.go:53: PutBucketLifecycle Success!
	lifecycle_test.go:63: GetBucketLifecycle Success!
	lifecycle_test.go:73: GetObject before lc Success value: valueput
	lifecycle_test.go:86: lc Success!
	lifecycle_test.go:99: GetObject after lc test Fail! <nil>
test cases still have the following fails.

**Comment by sunfengyun** [comment]
@thesues 
--- PASS: Test_LifeCycle (3.61s)
	lifecycle_test.go:22: CreateBucket Success!
	lifecycle_test.go:29: PutObject Success!
	lifecycle_test.go:53: PutBucketLifecycle Success!
	lifecycle_test.go:63: GetBucketLifecycle Success!
	lifecycle_test.go:73: GetObject before lc Success value: valueput
	lifecycle_test.go:86: lc Success!
	lifecycle_test.go:94: GetObject after lc test Success: NoSuchKey: The specified key does not exist.
			status code: 404, request id: , host id: 
	lifecycle_test.go:110: DeleteBucketLifecycle Success!
	lifecycle_test.go:117: DeleteBucket Success.

File: integrate/yigconf/yig.toml [hunk 1]
```diff
@@ -11,6 +11,7 @@ ssl_key_path = ""
 ssl_cert_path = ""
 
 # DebugMode
+lcdebug = true
```
Comment (chenji-kael) [inline integrate/yigconf/yig.toml]: This config should be set as flase by default.

File: test/go/lifecycle_test.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                t.Fatal("change dir in lc err:", err)
+        }
+	cmd := exec.Command("bash", "runlc.sh")
+	err = cmd.Run()
+	if err != nil {
+		t.Fatal("lc err:", err)
+	}
+	time.Sleep(time.Second * 3)
+	t.Log("lc Success!")
+    os.Chdir("../test/go")
```
Comment (cxt90730) [inline test/go/lifecycle_test.go]: It is not necessary

File: integrate/yigconf/yig.toml (hunk 1 above)
Comment (sunfengyun) [reply integrate/yigconf/yig.toml]: The config that you specified has been deleted as same as before.

Comment (sunfengyun) [reply integrate/yigconf/yig.toml]: The config you specified has been deleted and remains the same as before.

**Comment by sunfengyun** [comment]
@cxt90730  I have pulled my request to other branch ('develop') as you specified.

**Pull request closed without merging**